homeassistant/components/ios.py

```diff
@@ -136,7 +136,7 @@
 IDENTIFY_DEVICE_SCHEMA_CONTAINER = vol.All(dict, IDENTIFY_DEVICE_SCHEMA)
 
 IDENTIFY_APP_SCHEMA = vol.Schema({
-    vol.Required(ATTR_APP_BUNDLE_IDENTIFER): cv.string,
+    vol.Required(ATTR_APP_BUNDLE_IDENTIFIER): cv.string,
```

Comment:
Breaking change - `bundleIdentifer` -> `bundleIdentifier` - this doesn't seem to be used anywhere in code though.